x-pack/plugins/reporting/common/constants.ts

```diff
@@ -25,6 +25,10 @@ export const ALLOWED_JOB_CONTENT_TYPES = [
   'text/plain',
 ];
 
+export const EVENT_ACTION_EXECUTE_START = 'execute-start';
+export const EVENT_ACTION_EXECUTE_COMPLETE = 'execute-complete';
+export const EVENT_ACTION_EXECUTE_ERROR = 'execute-error';
```

Comment:
In Reporting, we can encounter errors but still come up with a report. For example, a dashboard could have a single panel that is throwing an error - but we will still take the screenshot of what is available. Or CSV export could encounter errors in building some rows - but we will still export as many rows as we can.  @pmuellr I think that makes a case for having a separate event for error.

Reply:
Ya, could be in this case having separate "error" events will work out fine.  Kinda depends on how you want to search for them.  For instance, if you generate one event for each individual error, then you'll be able to agg over those.  It would be harder/impossible if the errors were embedded as an array of string (for example) in a single event.

Reply:
I doubled back on this thinking and am going with the suggestion to NOT have an `execute-error` action type